src/dos/program_imgmount.h

```diff
@@ -25,6 +25,7 @@
 
 class IMGMOUNT final : public Program {
     public:
+        void ListImgMounts(void);
```

Comment:
very minor style change.. we always knockout `(void)` for `()` when possible. 
`void ListImgMounts();`, because it matches how the caller uses it, and is a bit easy to read.

Reply:
Done.